stream_alert/alert_processor/output_base.py

```diff
@@ -55,6 +55,10 @@ class StreamOutputBase(object):
     __metaclass__ = ABCMeta
     __service__ = NotImplemented
 
+    # _DEFAULT_REQUEST_TIMEOUT indicates long the requests library will wait before timing
```

Comment:
This should be `indicates how long` not `indicates long` 🤣

Reply:
@javuto before merging can you also update the comment where this was copied from?